web_modules/GoogleAnalyticsTracker/index.js

```diff
@@ -4,16 +4,31 @@ import { PropTypes } from "react"
 import ga from "react-google-analytics"
 const GoogleAnalyticsInitiailizer = ga.Initializer
 
-class GoogleAnalyticsTracker extends Component {
+const isProduction = process.env.NODE_ENV === "production"
+const isClient = typeof window !== "undefined"
+
+export default class GoogleAnalyticsTracker extends Component {
```

Comment:
Sorry I just copy pasted this from original statinamic snippet. Will revert this.